### PR TITLE
tck tests

### PR DESCRIPTION
**Pull request by AviAvni**
fixes https://github.com/RedisGraph/RedisGraph/issues/2262

### CODE REVIEW

File: tests/tck/utils/assertions.py [hunk 1]
```diff
@@ -189,13 +189,18 @@ def assert_statistics(resultset, stat, value):
...
+    elif stat == "-relationships":
+        Env.RTestInstance.currEnv.assertEquals(resultset.relationships_deleted, value)
     elif stat == "+labels":
         Env.RTestInstance.currEnv.assertEquals(resultset.labels_added, value)
     elif stat == "+properties":
         Env.RTestInstance.currEnv.assertEquals(resultset.properties_set, value)
     elif stat == "-nodes":
         Env.RTestInstance.currEnv.assertEquals(resultset.nodes_deleted, value)
+    elif stat == "-labels" or stat == "-properties":
+        pass
```
Comment (swilly22) [inline tests/tck/utils/assertions.py:201]: This has the potential of hiding incorrectness

**Review by swilly22 — APPROVED** [review]
great to see additional TCKs being active

**Comment by swilly22** [comment]
Seems like we're leaking

File: tests/tck/utils/assertions.py (hunk 1 above)
Comment (AviAvni) [reply tests/tck/utils/assertions.py:201]: we don't report about this stats
in the Scala check this is not checked by stats returned from the query

**Pull request merged**